src/actions/entries.js

```diff
@@ -265,9 +266,23 @@ export function persistEntry(collection) {
   return (dispatch, getState) => {
     const state = getState();
     const entryDraft = state.entryDraft;
+    const fieldsErrors = entryDraft.get('fieldsErrors');
 
     // Early return if draft contains validation errors
-    if (!entryDraft.get('fieldsErrors').isEmpty()) return Promise.reject();
+    if (!fieldsErrors.isEmpty()) {
+      const hasPresenceErrors = fieldsErrors
+        .some(errors => errors.some(error => error.type && error.type === ValidationErrorTypes.PRESENCE));
```

Comment:
Is there a reason we only check for `PRESENCE` errors here, instead of all the kind of Validation errors? That's obviously the actual issue request, but why not make a popup for pattern or custom errors as well?

CC/ @erquhart

Reply:
I [originally had it working this way](https://streamable.com/2evve) as it was a lot easier than introducing types for the different errors and interrogating errors based on type when saving.

I'm happy with either approach but have a slight preference towards the original as it both addresses the issue by identifying required fields that failed validation and is a much smaller change. 👍